.github/workflows/ci.yml

```
@@ -64,7 +66,7 @@ jobs:
     env:
       ALLOW_PLOTTING: 'true'
       DISPLAY: ':99'
-      PYTHON_VERSION: '3.8'
+      PYTHON_VERSION: '3.9'
```

Trying to revert the following:

```suggestion
      PYTHON_VERSION: '3.8'
```